tests/unit/Form/Block/BaseTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace MailPoet\Test\Form\Block;
+
+use MailPoet\Form\Block\Base;
+use MailPoet\Form\Util\FieldNameObfuscator;
+use MailPoet\WP\Functions as WPFunctions;
+use PHPUnit\Framework\MockObject\MockObject;
+
+class BaseTest extends \MailPoetUnitTest {
+  /** @var Base */
+  private $baseRenderer;
+
+  /** @var MockObject|WPFunctions */
```

Comment:
I noticed you separated the `|` operator by space from both sides in other tests. You don't do that here, is there a reason for that?